Snakemake/workflow/Snakefile

```diff
@@ -27,17 +27,27 @@ report: "reports/tsinfer.rst"
 # (2) sort into the correct Chr folder in vcfDir.
 
 # defining variables that will be used throughout the script
-vcfdir=config['vcf_dir'] # from config file, the path to vcf
+
+# the final directory name is now editable -- so can be named according to specific project
+# mapdir indicates where to find the genetic map for phasing (full path in the config)
+Project = config['PROJECT']
```

Comment:
Are you using this anywhere else?

Reply:
yes, just added a line in the Snakefile.

Reply:
It's used only once ATM:
```
Project = config['PROJECT']
mapdir = config['genmap']
oDir = Path(config['o_dir'], Project)
```
So, could replace by:
```
mapdir = config['genmap']
oDir = Path(config['o_dir'], config['PROJECT'])
```